ocean.js

```diff
@@ -0,0 +1,425 @@
+  death();
+}
+
+// Beach room
+function beach() {
+  var currentSituation = alert("As you swim towards the right, a wave comes from behind you and you ride it all the way to the beach. You crash into the sand and after a moment of confusion, you pick yourself up and look around.")
+  var move =  prompt("A man appoaches you and asks you to pick a number. '1' or '2'. Confused, you are not sure what to do. You can either 'run' or pick a number.");
+  // Checking moves
+  if (move === 'exit') {
+    return;
+  } else if (move === 'run') {
+    blackFishDeath();
+  } else if (parseInt(move) === 1) {
+    beachFood();
+  } else if (parseInt(move) === 2) {
+    beachDeath();
+  } else {
+    unknownMove();
+    beach();
+  };
+};
+
+// Choice for choosing number one which puts you in a groundhogs day loop
+function beachFood() {
+  alert("Pleased with your choice, the man leads you up the beach to small hut. There he as food and wine and together you drink the night way. As the sun begins to rise the man offers you one last drink. You glup in down with pleasure and you start to feel dizzy. The man begins to laugh and says 'Welcome to the Ocean.' You collaps on the ground.")
+  alert("You have just woken up to find yourself floataing in the middle of the ocean. It is the dead of night, but the glow of moon allows you to see a few feet infront of you.")
+  ocean();
+}
+
+// death from choosing to run
+function blackFishDeath() {
+  alert("As you begin to run the man looks disapointed and pulls out a whisle and blows it. Out of no where a Killer Whale glides up out of the ocean, grabs you, and pulls you back into the water.  The Killer Whales takes you back to his pod where he uses you as a toy to train some of the younger members of his pod. You survive for a while but are in horrible pain. When the whales are done with you they let you go and you sink to the bottom of the ocean.")
+  death();
+};
+
+// Death on beach from choosing number 2
+function beachDeath() {
+  alert("The man looks disapointed. He says 'Lucky You' and pulls out a pistol and shoots you in the head.")
+  death();
+}
+
+// Death prompt to see if they would like to play again
+function death() {
+  move = prompt("Looks like you have died. Would you like to play again? 'yes' or 'no'");
+  if (move === 'yes') {
+    ocean();
+  } else if (move === 'no') {
+    return;
+  } else {
+    unknownMove();
+    death();
+  };
+};
+
+function game() {
+  alert("Welcome to the Ocean. I hope you survive. Valid moves are placed between sinlge quotes(' '). If you would like to exit at any time, type 'exit'. There are hidden paths that are not specified moves. To find them try look around some times.");
+  var start = alert("You have just woken up to find yourself floating in the middle of the ocean. It is the dead of night, but the glow of moon allows you to see a little bit in front of you.");
+  ocean();
+};
+
+game();
```

Comment:
Great job, really like the aesthetic as well. Don't have much feedbacl; you nailed the lesson requirements, encapsulated choices in functions. If you wanted to take this further, you could store the control flow in an object. 💭 

👍